app/scripts/views/components/barChart/barChart.js

```diff
@@ -287,20 +322,26 @@
         }
       }
       chartInst_.render();
+
+      initTsvDownloadData();
+      initCanvasDownloadData();
```

Comment:
Why to initialize download data by default? Why don't we do it at the time of click event.